### PR TITLE
Third party code

### PR DESCRIPTION
**Pull request by Antarus66**
Added an intergration with CodePen and Pastebin

### CODE REVIEW

File: app/Http/Controllers/API/CodeSnippetController.php [hunk 1]
```diff
@@ -18,29 +20,99 @@ class CodeSnippetController extends Controller
...
+
+        try {
+            $data = $grabber->getRaw($link);
+        } catch (RemoteDataGrabberException $e) {
+            return $this->returnWidgetError('Pastebin snippet is not found');
+        }
+
+        // Getting a prefix and jsPath
+        $prefix = env('SERVER_PREFIX', '');
+        $jsPath = url(($prefix ? '/' : '' ) . env('JS_PATH'));
```
Comment (DevilRep) [inline app/Http/Controllers/API/CodeSnippetController.php]: Maybe we will add a function?

File: resources/views/errors/gist/404.blade.php [hunk 2]
```diff
@@ -7,8 +7,8 @@
     </head>
     <body>
         <div class="wrapper">
-            <img src="{{ $prefix }}/img/404-code.jpg" height="165" width="165">
-            <h1>GitHub Gist is not found</h1>
+            <img src="{{ url("img/404-code.jpg") }}" height="165" width="165">
```
Comment (DevilRep) [inline resources/views/errors/gist/404.blade.php]: Why you changed path to picture?

File: app/Http/Controllers/API/CodeSnippetController.php (hunk 1 above)
Comment (Antarus66) [inline app/Http/Controllers/API/CodeSnippetController.php]: I thought about it, but if we isolate a validation, there the same amount of code, so no sense. As for jsPath, I added one.

File: resources/views/errors/gist/404.blade.php (hunk 2 above)
Comment (Antarus66) [inline resources/views/errors/gist/404.blade.php]: Cause it's another try to make it work without a terrible variable in env.

Comment (DevilRep) [inline resources/views/errors/gist/404.blade.php]: We need to talk about it tomorrow

Comment (Antarus66) [inline resources/views/errors/gist/404.blade.php]: Sure. And about links to the pictures in .env too)

**Pull request merged**